<launch>
   <arg name="traj_drone1" default="$(find bridge_px4)/trajectories/line_grp_drone1.csv"/>
   <arg name="traj_drone2" default="$(find bridge_px4)/trajectories/line_grp_drone2.csv"/>
   <arg name="traj_drone3" default="$(find bridge_px4)/trajectories/line_grp_drone3.csv"/>
   <arg name="traj_drone4" default="$(find bridge_px4)/trajectories/line_grp_drone4.csv"/>
   <arg name="traj_drone5" default="$(find bridge_px4)/trajectories/line_grp_drone5.csv"/>
   <arg name="traj_drone6" default="$(find bridge_px4)/trajectories/line_grp_drone6.csv"/>

   <arg name="t_fs" default="30"/>
   <arg name="err_tol" default="0.225"/>

   <group ns="drone1">
   	<node pkg="bridge_px4" type="sp_pos_t_node" name ="sp_pos_t_node" output="screen">
           <param name="t_fs" value="$(arg t_fs)"/>
	   <param name="err_tol" value="$(arg err_tol)"/>
	   <param name="traj_id" value="$(arg traj_drone1)"/>
	</node>
   </group>

   <group ns="drone2">
   	<node pkg="bridge_px4" type="sp_pos_t_node" name ="sp_pos_t_node" output="screen">
         <param name="t_fs" value="$(arg t_fs)"/>
	      <param name="err_tol" value="$(arg err_tol)"/>
	      <param name="traj_id" value="$(arg traj_drone2)"/>
	</node>
   </group>

   <group ns="drone3">
   	<node pkg="bridge_px4" type="sp_pos_t_node" name ="sp_pos_t_node" output="screen">
         <param name="t_fs" value="$(arg t_fs)"/>
	      <param name="err_tol" value="$(arg err_tol)"/>
	      <param name="traj_id" value="$(arg traj_drone3)"/>
	</node>
   </group>

   <group ns="drone4">
   	<node pkg="bridge_px4" type="sp_pos_t_node" name ="sp_pos_t_node" output="screen">
         <param name="t_fs" value="$(arg t_fs)"/>
	      <param name="err_tol" value="$(arg err_tol)"/>
	      <param name="traj_id" value="$(arg traj_drone4)"/>
	</node>
   </group>

   <group ns="drone5">
   	<node pkg="bridge_px4" type="sp_pos_t_node" name ="sp_pos_t_node" output="screen">
         <param name="t_fs" value="$(arg t_fs)"/>
	      <param name="err_tol" value="$(arg err_tol)"/>
	      <param name="traj_id" value="$(arg traj_drone5)"/>
	   </node>
   </group>

   <group ns="drone6">
   	<node pkg="bridge_px4" type="sp_pos_t_node" name ="sp_pos_t_node" output="screen">
         <param name="t_fs" value="$(arg t_fs)"/>
	      <param name="err_tol" value="$(arg err_tol)"/>
	      <param name="traj_id" value="$(arg traj_drone6)"/>
	</node>
   </group>
</launch>
